{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "children": [],
        "cpe_match": [
          {
            "cpe23Uri": "cpe:2.3:a:tigervnc:tigervnc:-:*:*:*:*:*:*:*",
            "cpe_name": [],
            "vulnerable": true
          }
        ],
        "operator": "OR"
      },
      {
        "children": [],
        "cpe_match": [
          {
            "cpe23Uri": "cpe:2.3:a:x.org:x_server:-:*:*:*:*:*:*:*",
            "cpe_name": [],
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:a:x.org:xwayland:-:*:*:*:*:*:*:*",
            "cpe_name": [],
            "vulnerable": true
          }
        ],
        "operator": "OR"
      },
      {
        "children": [],
        "cpe_match": [
          {
            "cpe23Uri": "cpe:2.3:o:redhat:enterprise_linux:7.0:*:*:*:*:*:*:*",
            "cpe_name": [],
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:o:redhat:enterprise_linux:8.0:*:*:*:*:*:*:*",
            "cpe_name": [],
            "vulnerable": true
          },
          {
            "cpe23Uri": "cpe:2.3:o:redhat:enterprise_linux:9.0:*:*:*:*:*:*:*",
            "cpe_name": [],
            "vulnerable": true
          }
        ],
        "operator": "OR"
      }
    ]
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "secalert@redhat.com",
      "ID": "CVE-2025-26600"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "A use-after-free flaw was found in X.Org and Xwayland. When a device is removed while still frozen, the events queued for that device remain while the device is freed. Replaying the events will cause a use-after-free."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": []
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://access.redhat.com/security/cve/CVE-2025-26600",
          "refsource": "",
          "tags": [
            "Third Party Advisory"
          ],
          "url": "https://access.redhat.com/security/cve/CVE-2025-26600"
        },
        {
          "name": "RHBZ#2345252",
          "refsource": "",
          "tags": [
            "Issue Tracking"
          ],
          "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2345252"
        }
      ]
    }
  },
  "impact": {},
  "lastModifiedDate": "2025-03-04T17:22Z",
  "publishedDate": "2025-02-25T16:15Z"
}